src/components/authButtons.tsx

```diff
@@ -4,5 +4,9 @@ import { signOut } from "next-auth/react";
 import { type PropsWithChildren } from "react";
 
 export function SignOutButton({ children }: PropsWithChildren) {
-  return <span onClick={() => signOut()}>{children}</span>;
+  return (
+    <span onClick={() => signOut({ redirect: true, callbackUrl: "/" })}>
```

Comment:
I think redirect only works, when we have a Credentials provider in auth.js. That is a minor issue and can be ommited.

Reply:
Will me omitted